torch/_inductor/utils.py

```diff
@@ -680,6 +680,13 @@ def has_incompatible_cudagraph_ops(gm):
     return False
 
 
+def output_node(gm: torch.fx.GraphModule):
+    """Get the output node from an FX graph"""
+    last_node = next(iter(reversed(gm.graph.nodes)))
+    assert last_node.op == "output"
+    return last_node
```

Comment:
@jansel weren't you telling us about how sometimes the output node isn't last lol

Reply:
I think others were saying these asserts fail.  I think we should enforce this constraint.